gilded-rose-llc-frontend/pages/index.tsx

```diff
@@ -0,0 +1,46 @@
+import type { NextPage } from "next";
+import Head from "next/head";
+
+interface ItemBlockProps {
+  name: string;
+  price: string;
+}
+
+const ItemBlock = (props: ItemBlockProps) => {
```

Comment:
Are we expecting this to be a placeholder or is this going to be the front page? If this is just a placeholder, then it's probably fine, otherwise, we should have tests for this.

Reply:
This was just a placeholder to ensure Tailwind was working properly. Our next PR is going to cover the actual implementation of the page with tests